support/Environments/frontera_gcc.sh

```diff
@@ -252,12 +70,15 @@ spectre_run_cmake() {
         return 1
     fi
     spectre_load_modules
+    # -D USE_LD=ld - ld.gold seems to hang linking the main executables
     cmake -D CHARM_ROOT=$CHARM_ROOT \
```

Comment:
I had to add `DEBUG_SYMBOLS=OFF` to fit the build of executables into memory on the head node (which I probably shouldn't have done). Maybe add a note to compile on a compute node, along with the command to allocate one? And/or turn `DEBUG_SYMBOLS` off?